client/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "react-webpack-rails-tutorial",
+  "version": "1.1.0",
+  "description": "Built using the react_on_rails generator.",
+  "main": "server.js",
+  "engines": {
+    "node": "5.0.0",
+    "npm": "3.5.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial.git"
+  },
+  "keywords": [
+    "react",
+    "tutorial",
+    "comment",
+    "example"
+  ],
+  "author": "justin808",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial/issues"
+  },
+  "homepage": "https://github.com/shakacode/react-webpack-rails-tutorial",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js",
+    "build:client": "NODE_ENV=production webpack --config webpack.client.rails.config.js",
+    "build:server": "NODE_ENV=production webpack --config webpack.server.rails.config.js",
+    "build:dev:client": "webpack -w --config webpack.client.rails.config.js",
+    "build:dev:server": "webpack -w --config webpack.server.rails.config.js",
+    "lint": "npm run eslint && npm run jscs",
+    "eslint": "eslint --ext .js,.jsx .",
+    "jscs": "jscs --verbose ."
+  },
+  "dependencies": {
+    "babel-core": "^5.8.25",
+    "babel-loader": "^5.3.2",
+    "body-parser": "^1.14.1",
+    "es5-shim": "^4.3.1",
+    "es6-promise": "^3.0.2",
+    "expose-loader": "^0.7.1",
+    "immutable": "^3.7.5",
+    "imports-loader": "^0.6.5",
+    "jquery": "^2.1.4",
+    "jquery-ujs": "^1.1.0-1",
+    "loader-utils": "^0.2.11",
+    "lodash": "^3.10.1",
+    "mirror-creator": "0.0.1",
+    "react": "^0.14.3",
+    "react-bootstrap": "^0.28.1",
+    "react-dom": "^0.14.3",
+    "react-redux": "^4.0.0",
+    "redux": "^3.0.4",
+    "redux-promise": "^0.5.0",
+    "redux-thunk": "^1.0.0",
+    "webpack": "^1.12.8"
```

Comment:
**NOTE**: We have to put any tools used for building the bundles in `dependencies` and **NOT** `devDependencies` as Heroku will only load the `dependencies` npm packages.